deploy/ansible/roles/legion_enclave/templates/airflow_s3_access_policy.yaml.j2

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+     {
+      "Effect": "Allow",
+      "Action": ["s3:*"],
+      "Resource": [
+        "arn:aws:s3:::{{ airflow_s3_bucket_name }}/{{ enclave }}",
```

Comment:
Is it airflow log bucket or data bucket ?

Reply:
it's a path that this Airflow can access. It's used for data and logs